Patched to look for the .csv file in /usr/share/exploitdb
--- a/searchsploit
+++ b/searchsploit
@@ -13,7 +13,7 @@
 
 
 ## OS settings (get the path of where the script is stored + database file)
-gitpath="/opt/exploit-database"
+gitpath="/usr/share/exploitdb"
 csvpathexploits="${gitpath}/files_exploits.csv"
 csvpathshellcode="${gitpath}/files_shellcodes.csv"
 
@@ -751,4 +751,4 @@ fi
 
 
 ## Done
-exit 0
\ No newline at end of file
+exit 0
